pom.xml

```diff
@@ -133,6 +133,17 @@
                 </executions>
             </plugin>
         </plugins>
+        <resources>
+            <!-- Apply the properties set in the POM to the resource files -->
+            <resource>
+                <filtering>true</filtering>
+                <directory>src/main/resources</directory>
```

Comment:
I think src/main/resources is the default value for directory, no? It shouldn't be needed to specify it.

Reply:
`directory` config is needed. After deleting it, you get:

```
[ERROR] 'build.resources.resource.directory' is missing. @ line 138, column 23
```